config/prod.exs

```diff
@@ -28,6 +28,8 @@ config :blockchain_api,
   google_maps_secret: System.get_env("GOOGLE_MAPS_API_KEY"),
   onesignal_rest_api_key: System.get_env("ONESIGNAL_API_KEY"),
   onesignal_app_id: System.get_env("ONESIGNAL_APP_ID"),
+  fastly_api_key: System.get_env("FASTLY_API_KEY"),
```

Comment:
We should may be put these in `dev.exs` as well for alamo support?

Reply:
oh hmm does alamo run in dev environment? that seems a bit odd, but yeah I can add it to dev as well. we just wouldn't set those env vars when actually running in dev, but the module's designed to handle that case

Reply:
Yeah, had to separate it out to not do notifications from alamo.